### ISSUE TITLE
Monochrome icon missing for GreenStash app

### ISSUE THREAD

**Issue by D4rK7355608** [Pool-Of-Tears/GreenStash#21]
The current version of the GreenStash app does not have a monochrome icon, which is causing some accessibility and design issues for users. They also help maintain a consistent design language throughout the app.

Steps to Reproduce:

1. Install the latest version of the GreenStash app on any device.
2. View the app icon on the home screen or app drawer.
3. Notice that the icon is in full color and not monochrome.

Expected Behavior:
The GreenStash app should have a monochrome icon in addition to the full-color icon. The monochrome icon should be easily distinguishable and reflect the design language of the app.

Actual Behavior:
The GreenStash app does not have a monochrome icon, which is causing accessibility and design issues for some users.

Reproducibility:
This issue can be reproduced on any device that has the latest version of the GreenStash app installed.

Suggested Solution:
A monochrome icon should be added to the GreenStash app to improve accessibility and maintain a consistent design language. The icon should be easily distinguishable and reflect the design of the app.

**Comment by starry-shivam** [comment]
Hi! first of all thanks for creating such a detailed issue, apparently I'm planning to rewrite this whole app in jetpack compose (see #18 ), I'll try adding support for Android 13 monochrome icon after I'm done with that. Will keep the issue open until then for the reminder.

**Comment by starry-shivam** [comment]
Closing as monochrome icon has been added in the last commit and will be available in the next release (whenever that happens)
